test/accel/HepMC3PrimaryGenerator.test.cc

```diff
@@ -148,7 +148,7 @@ TEST_F(HepMC3PrimaryGeneratorTest, no_vertex)
         999.99998921041, 1000, 999.99998921041, 1000.0000107896,
         999.99998921041, 999.99998921041, 1000, 1000, 999.99998921041,
         1000.0000107896, 1000.0000107896, 999.99998474121,};
-    EXPECT_VEC_SOFT_EQ(expected_energy, result.energy);
+    EXPECT_VEC_NEAR(expected_energy, result.energy, coarse_eps);
```

Comment:
What caused this change?

Reply:
The `develop` branch fails on my local machine (M2 ARM): I guess this calculation not only introduces 7 digits of roundoff error, but the roundoff is quite sensitive to the architecture.